docs/openapi/api-eservice-operations-interop-v1.yaml

```diff
@@ -678,6 +723,22 @@ components:
         producerName:
           title: Service producer name
           type: string
+    
+    ProbingDataEserviceResponse:
+      type: object
+      properties:
+        probingEnabled:
```

Comment:
IMO this boolean is something related to the presentation layer, and should not be calculated here.
The `operations` layer should return data as close as possible to the persistence, doing transformations only when absolutely required (e.g. when querying by `EserviceMonitorState`)

Reply:
There is already a technical debt in the backlog regarding moving the presentational logic from operations to probing-api.Maybe we can add this part to that?

Reply:
the creation is in this PR, I don't see the point in doing and undoing it

Reply:
ok